code/datums/outfits/outfit_admin.dm

```diff
@@ -220,24 +220,40 @@
 
 /datum/outfit/admin/nt_undercover
 	name = "NT Undercover Operative"
```

Comment:
Maybe if you're creating a lesser version, use 'Agent' instead of 'Operative'.
Rationale is that the syndicate already have a similar structure ("syndicate agents" are undercover players on station, "syndicate officers" are admin-only event roles). So it makes it easier to understand what each does, and the difference between them. Using 'Agent' instead of officer also prevents both of them having the same abbreviation ("NTUO").

Reply:
I didn't think about the abbreviation thing, will go ahead and change it.